### DISCUSSION TITLE
DataCap supported GreptimeDB for the first time

### DISCUSSION THREAD

**Discussion by qianmoQ** [GreptimeTeam/greptimedb#1382] [General]
[[Plugin] [HTTP] Support greptimedb](https://github.com/EdurtIO/datacap/pull/320)

Welcome to use it to help us support more integration of GreptimeDB

**Comment by killme2008** [comment]
@qianmoQ Looks great, but the link is broken, can you fix it?

**Reply by qianmoQ** [reply]
Forgot to make PR, it has been fixed
